velox/exec/HashTable.h

```diff
@@ -33,7 +33,6 @@ struct HashLookup {
     rows.resize(size);
     hashes.resize(size);
     hits.resize(size);
```

Comment:
Let's add a comment here to explain that we intentially do not re-initialize hits with nulls.

Reply:
Will that be a bit "extra"? 
From readability pov, future readers will just assume it's like rows, hashes, simply allocated here but populated later during the course of lookup, which would be the correct assumption.

Reply:
Ok.